fishy/engine/semifisher/fishing_mode.py

```diff
@@ -1,45 +1,46 @@
 import time
-from enum import Enum
 
 subscribers = []
 
 
-class State(Enum):
-    IDLE     = [  0,   0, 255]
-    LOOKAWAY = [150, 255,  76]
-    LOOKING  = [100, 255, 101]
-    DEPLETED = [ 30, 255,  76]
-    NOBAIT   = [ 96, 255, 255]
-    FISHING  = [ 18, 165, 213]
-    REELIN   = [ 60, 255, 204]
-    LOOT     = [  0, 255, 204]
-    INVFULL  = [  0, 255,  51]
-    FIGHT    = [120, 255, 204]
-    DEAD     = [  0,   0,  51]
+State = {
+    "IDLE"     : [255, 255, 255],
+    "LOOKAWAY" : [ 76,   0,  76],
+    "LOOKING"  : [101,  69,   0],
+    "DEPLETED" : [  0,  76,  76],
+    "NOBAIT"   : [255, 204,   0],
+    "FISHING"  : [ 75, 156, 213],
+    "REELIN"   : [  0, 204,   0],
+    "LOOT"     : [  0,   0, 204],
+    "INVFULL"  : [  0,   0,  51],
+    "FIGHT"    : [204,   0,   0],
+    "DEAD"     : [ 51,  51,  51]
+}
```

Comment:
instead of using string, we can use enums as there are limited number of states

Reply:
I am not sure what you are saying.
The enum, as it was before, is static. It does not allow changing the state rgb values at runtime.

Reply:
use dictionary to store the rgb values, but u can make the key of State variable enum instead of string

Reply:
Hm okay, I think I know what you mean, but I am unsure how to create the dict then.